Makefile

```diff
@@ -156,7 +156,7 @@ tmp/$(HOST)-$(PORT).key tmp/$(HOST)-$(PORT).crt: ## Self-signed cert for local H
 run: ## Runs the development server
 	foreman start -p $(PORT)
 
-urn:
+urn: ## A little treat if you ever mistype "run" ⚱️
```

Comment:
Tempting to keep this undocumented / secret 🤫

Reply:
```suggestion
urn:
```

Reply:
ok! it's gone in 36e4e2a870139dd12dbd3e82b7bcf08225edecb7